libs/hmpb-interpreter/test/mockOf.ts

```diff
@@ -1,15 +0,0 @@
-/**
- * Returns a properly-typed mock for an already-mocked function.
- *
- * @example
- *
- * import * as fs from 'fs'
- * jest.mock('fs')
- * const readFileMock = mockOf(fs.readFile)
- * readFileMock.mockImplementation(…)
- */
-export default function mockOf<T extends (...args: unknown[]) => unknown>(
-  fn: T
-): jest.MockedFunction<T> {
-  return fn as jest.MockedFunction<T>
-}
```

Comment:
The newly-expanded `collectCoverageFrom` value in the shared config flagged this as totally uncovered, and since it's only ever used in tests I just deleted it.